spec/features/events_spec.rb

```diff
@@ -72,8 +74,13 @@
 
     click_link "Death Star's destruction celebration"
     page.should have_link('Email')
+    page.should have_link('Participants')
 
     click_link 'Email'
     page.should have_content "Emails for Death Star's destruction celebration"
+
+    visit event_path(events(:party))
+    click_link 'Participants'
+    page.should have_content "Participants of Death Star's destruction celebration"
```

Comment:
The test Christian asked for. To test that a user can not see things he is not entitled to